config/locales/en/en.yml

```diff
@@ -164,6 +164,8 @@ en:
     labels:
       timeline_step_move:
         move_type: "Which Date?"
+      accounting_transaction:
+        private_note: 'Memo'
```

Comment:
`en.activerecord.attributes` has path of `attributes.accounting.transaction`.

The form is expecting the path of `accounting_transaction`.